crates/ruma-events/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ Bug fixes:
 
 Breaking changes:
 
+- Rename the `key` field in the `SecretStorageDefaultKeyEventContent` to
+  `key_id`.
+
```

Comment:
It's a bit weird that there's a newline here.

Reply:
Fixed.